app/wear/src/main/kotlin/io/github/xxfast/krouter/wear/SwipeDismissContent.kt

```diff
@@ -0,0 +1,92 @@
+package io.github.xxfast.krouter.wear
+
+import android.os.Parcelable
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.SaveableStateHolder
+import androidx.compose.runtime.saveable.rememberSaveableStateHolder
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.wear.compose.foundation.ExperimentalWearFoundationApi
+import androidx.wear.compose.foundation.HierarchicalFocusCoordinator
+import androidx.wear.compose.material.MaterialTheme
+import androidx.wear.compose.material.SwipeToDismissBox
+import androidx.wear.compose.material.SwipeToDismissKeys
+import androidx.wear.compose.material.rememberSwipeToDismissBoxState
+import com.arkivanov.decompose.Child
+import com.arkivanov.decompose.ComponentContext
+import com.arkivanov.decompose.router.stack.ChildStack
+import com.arkivanov.decompose.router.stack.pop
+import io.github.xxfast.krouter.LocalComponentContext
+import io.github.xxfast.krouter.LocalRouter
+import io.github.xxfast.krouter.Router
+
+@OptIn(ExperimentalWearFoundationApi::class)
+@Composable
+fun <C : Parcelable> RoutedSwipeDismissContent(
+  router: Router<C>,
+  modifier: Modifier = Modifier,
+  content: @Composable (C) -> Unit,
+) {
+  val stack: ChildStack<C, ComponentContext> by router.stack
+  val active: Child.Created<C, ComponentContext> = stack.active
+  val background: Child.Created<C, ComponentContext>? = stack.backStack.lastOrNull()
+  val holder: SaveableStateHolder = rememberSaveableStateHolder()
+  holder.RetainStates(stack.getConfigurations())
+
+  // TODO fix in wear compose
```

Comment:
Hi @yschimke, is this related to https://issuetracker.google.com/issues/280392104?

Reply:
Yep, sorry forgot to follow up

Reply:
Fixed now


```
    androidx.wear.compose:compose-material:1.3.0-alpha01
```